ppdet/data/transform/keypoint_operators.py

```diff
@@ -0,0 +1,462 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# function:
+#    operators to process sample,
+#    eg: decode/resize/crop image
+
+from __future__ import absolute_import
+
+try:
+    from collections.abc import Sequence
+except Exception:
+    from collections import Sequence
+
+import cv2
+import numpy as np
+import math
+import copy
+import os
+
+from ppdet.core.workspace import serializable
+from ppdet.utils.logger import setup_logger
+logger = setup_logger(__name__)
+
+registered_ops = []
+
+__all__ = ['RandomAffine', 'KeyPoint_Flip', 'TagGenerate', 'ToHeatmaps', 'Normalize_Permute', 'EvalAffine', 'get_affine_transform', 'transpred']
+
+def register_keypointop(cls):
+    return serializable(cls)
+
+class ImageError(ValueError):
+    pass
+
+@register_keypointop
+class KeyPoint_Flip(object):
```

Comment:
1. KeyPoint_Flip -> KeyPointFlip
2. add comments
3. whether need to inherit BaseOperator in  transform/operators.py

Reply:
done,
re 3. not too much codes, rewrite is clear to read